src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__item calendar__item--start-day-wed calendar__item--month-length-29">
```

Comment:
it'd be better to follow BEM recommendations like this
```suggestion
    <div class="container">
      <div class="calendar calendar--start-day--wed calendar--month-length--29">
```

Reply:
but it was from the beginning. the previous mentor rejected and demanded a replacement.